web-client/src/components/MyReviewsOneReview.jsx

```diff
@@ -0,0 +1,122 @@
+import React from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import Button from '@material-ui/core/Button';
+import '../App.css';
+
+function MyReviewsOneReview(props)
+{
+    const styles = makeStyles({
+        main:
+        {
+            width: '100%',
+            paddingBottom: '300px',
+        },
+        h2:
+        {
+            width: '90%',
+            float: 'left',
+            textAlign: 'left',
+            margin: 'auto',
+            marginTop: '2%',
+        },
+        panel:
+        {
+            width: '100%',
+            float: 'left',
+        },
+        author:
+        {
+            float: 'left',
+            textAlign: 'left',
+            margin: 'auto',
+            marginTop: '1%',
+        },
+        photo:
+        {
+            float: 'left',
+            width: '6%',
+            height: '35px',
```

Comment:
Width depends on size of website, but height have a constant value. @upsu90, he should leave it for now?
![image](https://user-images.githubusercontent.com/15444187/102351916-7c27bf00-3fa7-11eb-9b3c-be16815a8377.png)
![image](https://user-images.githubusercontent.com/15444187/102351929-7fbb4600-3fa7-11eb-8afc-3cd9404b85dc.png)

Reply:
I did that all the time because height I can not describe in %

Reply:
I didn't notice this earlier :/ The best solution is min/max properties, I guess 😄 So, you can leave it. Later we'll change it

Reply:
it's best to leave it and then see how these avatars scale

Reply:
.... but it should look like this one: 
![image](https://user-images.githubusercontent.com/44305068/102544607-5688db80-40b5-11eb-8aa3-72985f9bc9a8.png)
we forgot to change it in miro ;-;

Reply:
I did now